src/api/property/float.js

```diff
@@ -0,0 +1,6 @@
+export default {
+  default: 0,
+  serialize: String,
+  deserialize: parseFloat,
```

Comment:
I was unsure of that when I was spiking them. I think that's probably a good idea. What are your thoughts?